src/styles/stars.css

```diff
@@ -0,0 +1,36 @@
+.product__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin: 0 0 24px;
+  line-height: 14px;
```

Comment:
seems like this has already been set in the html selector

Reply:
Thank you fixed